HouseRules_Essentials/Rules/PieceUseWhenKilledOverriddenRule.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using System.Collections.Generic;
+    using Boardgame;
+    using Data.GameData;
+    using DataKeys;
+    using HarmonyLib;
+    using HouseRules.Types;
+
+    public sealed class PieceUseWhenKilledOverriddenRule : Rule, IConfigWritable<Dictionary<BoardPieceId, List<AbilityKey>>>, IMultiplayerSafe
+    {
+        public override string Description => "Piece UseWhenKilled lists are overridden";
+
+        protected override SpecialSyncData ModifiedData => SpecialSyncData.StatusEffectImmunity;
```

Comment:
Great foresight.

Reply:
It would have been if I'd done that intentionally. When creating new rules, I generally copy/pasted a similar rule as a framework and then change bits as needed. I have now corrected this to use `SpecialSyncData.PieceData` instead of `SpecialSyncData.StatusEffectImmunity`.